.reek

```diff
@@ -44,6 +46,10 @@ UtilityFunction:
     - SessionTimeoutWarningHelper#frequency
     - SessionTimeoutWarningHelper#start
     - SessionTimeoutWarningHelper#warning
+    - SessionDecorator#registration_heading
+    - ServiceProviderSessionDecorator#registration_heading
+    - SessionDecorator#registration_bullet_1
+    - ServiceProviderSessionDecorator#registration_bullet_1
```

Comment:
Only the `SessionDecorator` class needs to be excluded here. The `ServiceProviderSessionDecorator` methods do use the instance variable `sp_name`.